Iconic/App_Plugins/Iconic/js/src/iconic.prevalues.editor.controller.js

```diff
@@ -136,7 +136,7 @@
         if (!item.sourcefile) item.sourcefile = item.cssfile;
 
 
-        var path = umbRequestHelper.convertVirtualToAbsolutePath("~" + item.sourcefile.replace("wwwroot",""));
+        var path = umbRequestHelper.convertVirtualToAbsolutePath("~/" + item.sourcefile.replace("wwwroot",""));
```

Comment:
hmmm, I have to take a look at this. My first version has ~/ but it didn't work so I removed it. Maybe this is not the correct way? Not sure why it can work on my local but not yours...

Reply:
That is interesting, Maybe different browsers hand over paths starting with a / and some don't? I am on Chrome 96.0.4664.45 (Win10).

Dirty fix would be to do 
`("~/" + item.sourcefile.replace("wwwroot","").replace("~//","~/)` (if the issue is double slashes)
But I do feel this kind of logic should be in a helper class (like Umbraco did with the umbRequestHelper)

Would try this out across multiple browsers, but in bed with a fever atm :(

Reply:
take care man, I hope your better ;)

I have been checking this path issue. This is why it's working for me:
![image](https://user-images.githubusercontent.com/1685262/143999349-053037a4-554d-45a3-b010-e0d80edf5284.png)

so if from the sourceFile path we remove the `wwwroot` we get `/css/file.css`. We can prepend just ~ before it. What path are you getting from the sourceFile?

Reply:
Starting to feel better, here you go

![image](https://user-images.githubusercontent.com/6025797/144225215-8f841b6e-c5d4-4e54-b2df-3343f93df3d5.png)

Reply:
Made a little fiddle with a helper function
https://jsfiddle.net/2cqp57y8/18/

Reply:
Thanks for that. I don't think we need to check if ~ is there or if there's double //. I reckon we should sanitize the value coming from sourceFile only and this value is returned by the . This value seems to be the path from the root, and it fails if it starts with `wwwroot`.

I have found that Umbraco is actually doing the same for the blocklist. They are replacing `wwwroot/` instead of just `wwwroot`. That might solve the problem.

![image](https://user-images.githubusercontent.com/1685262/144331647-d8889887-c613-4749-a08a-bf35bccc13ee.png)

Reply:
@Migaroez I have pushed a fix to master-v9, take a look. I reckon this will fix it for every possible path.

Reply:
Seems like you have a plan 👍